CHANGELOG.md

```diff
@@ -0,0 +1,71 @@
+# Changelog
+All notable changes to this project will be documented in this file.
```

Comment:
I'm afraid the wording is not quite right.
As far as I understand, https://jdbc.postgresql.org/ would still be the master source for the project. GitHub readme and changelog are the secondary sources for convenience purposes.

Reply:
The wording is from http://keepachangelog.com, I can move the wording of "Older releases" section, to the top and remove the default wording if you like.

Two "changelog" headers? do you mean the header on the middle of README.md and the header on the start of CHANGELOG.md?
I can remove all the reference of changelog in the README.md to keep things simple.

I see the https://jdbc.postgresql.org/ source as a historical point of view, more than a "master source", and in many ways, the CHANGELOG.md can be used as a source that feeds https://jdbc.postgresql.org/, the Unreleased section can be used to include changes in a PR, so when a release is ready, the changelog will be ready at the same time.

The point is to have a human readable list of changes, so if you think that the wording has to be changes let me know.